apps/ui/src/components/Ui/ProposalLabel.vue

```diff
@@ -46,7 +50,9 @@ function checkColorProximity(color: string): {
 </script>
 
 <template>
-  <div
+  <component
+    :is="to ? AppLink : 'div'"
+    :to="to"
```

Comment:
Maybe it would be better to handle this in AppLink. If `to` doesn't exist, it use `div` instead?